doc/classes/Vector3.xml

```diff
@@ -346,19 +349,19 @@
 			Enumerated value for the Z axis. Returned by [method max_axis] and [method min_axis].
 		</constant>
 		<constant name="ZERO" value="Vector3( 0, 0, 0 )">
-			Zero vector.
+			Zero vector, a vector with all components set to [code]0[/code].
 		</constant>
 		<constant name="ONE" value="Vector3( 1, 1, 1 )">
-			One vector.
+			One vector, a vector with all components set to [code]1[/code].
 		</constant>
 		<constant name="INF" value="Vector3( inf, inf, inf )">
-			Infinity vector.
+			Infinity vector, a vector with all components set to [code]INF[/code].
```

Comment:
If you decide to use `[constant @GDScript.INF]` in Vector2, make sure to do it here too.